### PR TITLE
#153114597 Send alerts via SMS

### PR DESCRIPTION
**Pull request by gr1d99**
#### What does this PR do?
It gives the ability to integrate sms alerts using `AfricasTalking Gateway`.

#### Description of Task to be completed?
should be able to send alerts via `SMS`.

#### How should this be manually tested?
Run the command below.

    $ python manage.py sendalerts

#### What are the relevant pivotal tracker stories?
[#153114597](https://www.pivotaltracker.com/n/projects/2131074/stories/153114597)

#### Screenshots (if appropriate)

screen shot 2017-12-20 at 18 16 43

----------------------------------------------------------------------------------

img-20171220-wa0000

### CODE REVIEW

File: templates/integrations/add_africastalking.html [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+            </div>
+        </form>
+    </div>
+</div>
+
+
+{% endblock %} {% block scripts %} {% compress js %}
+<script src="{% static 'js/jquery-2.1.4.min.js' %}"></script>
+<script src="{% static 'js/bootstrap.min.js' %}"></script>
+{% endcompress %} {% endblock %}
```
Comment (jonathankamau) [inline templates/integrations/add_africastalking.html]: New line required here.

Comment (silverjimmy) [reply templates/integrations/add_africastalking.html]: let me work on that and commit

File: templates/integrations/add_africastalking.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    </div>
+</div>
+{% endblock %} 
+
+{% block scripts %} 
+{% compress js %}
+<script src="{% static 'js/jquery-2.1.4.min.js' %}"></script>
+<script src="{% static 'js/bootstrap.min.js' %}"></script>
+{% endcompress %} 
+{% endblock %}
```
Comment (jonathankamau) [inline templates/integrations/add_africastalking.html]: There's no new line here.

Comment (jonathankamau) [inline templates/integrations/add_africastalking.html]: Add a new line here.

Comment (silverjimmy) [reply templates/integrations/add_africastalking.html]: Issue has been sorted

**Pull request merged**